plugin.video.svtplay/resources/lib/svtplay.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import absolute_import,unicode_literals
+import os
+import sys
+import xbmc # pylint: disable=import-error
+import xbmcaddon # pylint: disable=import-error
+import xbmcplugin # pylint: disable=import-error
+
+from resources.lib.settings import Settings
+from resources.lib.listing.router import Router
+from resources.lib import logging
+from resources.lib import helper
+
+class SvtPlay:
+
+    def __init__(self, plugin_handle, plugin_url, plugin_params):
+        self.addon = xbmcaddon.Addon("plugin.video.svtplay")
```

Comment:
This can just be `xbmcaddon.Addon()`. It will default to the current add-on. Not a show stopper, but perhaps for a next version?

Reply:
Thanks for the tip! 👍 
(and thanks for merging this!)